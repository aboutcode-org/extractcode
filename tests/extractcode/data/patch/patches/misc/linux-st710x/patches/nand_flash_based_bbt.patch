--- linux-2.6.23/drivers/mtd/nand/stm_nand_flex.c	2010-02-03 14:42:36.000000000 +0100
+++ nand_bbt_fix/stm_nand_flex.c	2010-02-03 14:45:21.000000000 +0100
@@ -883,10 +883,9 @@
 	data->chip.options |= NAND_NO_SUBPAGE_WRITE; /* Not tested, disable */
 	data->chip.options |= NAND_NO_AUTOINCR;      /* Not tested, disable */
 
-#ifdef CONFIG_STM_NAND_FLEX_BOOTMODESUPPORT
-	/* Force use of BBT when BOOTMODESUPPORT enabled */
+	/* We use Flash based BBT to save boot time */
 	data->chip.options |= NAND_USE_FLASH_BBT;
-#endif
+
 	/* Callbacks for FLEX mode operation */
 	data->chip.cmd_ctrl = flex_cmd_ctrl;
 	data->chip.select_chip = flex_select_chip;
