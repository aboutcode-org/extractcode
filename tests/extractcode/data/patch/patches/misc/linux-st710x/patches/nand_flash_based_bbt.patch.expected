[('linux-2.6.23/drivers/mtd/nand/stm_nand_flex.c',
  'nand_bbt_fix/stm_nand_flex.c',
  '--- linux-2.6.23/drivers/mtd/nand/stm_nand_flex.c\n+++ nand_bbt_fix/stm_nand_flex.c\n@@ -883,10 +883,9 @@\ndata->chip.options |= NAND_NO_SUBPAGE_WRITE; /* Not tested, disable */\ndata->chip.options |= NAND_NO_AUTOINCR;      /* Not tested, disable */\n\n-#ifdef CONFIG_STM_NAND_FLEX_BOOTMODESUPPORT\n-\t/* Force use of BBT when BOOTMODESUPPORT enabled */\n+\t/* We use Flash based BBT to save boot time */\ndata->chip.options |= NAND_USE_FLASH_BBT;\n-#endif\n+\n/* Callbacks for FLEX mode operation */\ndata->chip.cmd_ctrl = flex_cmd_ctrl;\ndata->chip.select_chip = flex_select_chip;')]